nautobot/utilities/tests/test_forms.py

```diff
@@ -429,3 +433,54 @@ def test_invalid_input(self):
         for test in tests:
             with self.assertRaises(forms.ValidationError):
                 self.field.clean(test)
+
+
+class AddressFieldMixinTest(TestCase):
+    """Test cases for the AddressFieldMixin."""
+
+    def setUp(self):
+        """Setting up shared variables for the AddressFieldMixin."""
+        self.ip = IPAddress.objects.create(address="10.0.0.1/24")
+        self.initial = {"address": self.ip}
```

Comment:
I would think this needs to be:

```suggestion
        self.initial = {"address": self.ip.address}
```

as IIRC `initial["address"]` should be a string, rather than an `IPAddress` instance. Same comment on Prefix test below. Sorry to be less than clear on my previous round of comments!